package.json

```diff
@@ -9,7 +9,8 @@
     "logo": "https://opencollective.com/opencollective/logo.txt"
   },
   "dependencies": {
-    "prop-types": "^15.6.2"
+    "deprecated-react-native-prop-types": "^2.3.0",
```

Comment:
The `yarn.lock` change needs to be committed with this since CI uses `yarn --frozen-lockfile`. It won't build otherwise.